Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM maven:latest
+
+WORKDIR /code
+
+COPY . /code
+
+ARG DB_HOST
+ARG DB_PASSWORD
+ARG DB_USERNAME
+ARG DB_NAME
+
+ENV DB_HOST ${DB_HOST}
+ENV DB_PASSWORD ${DB_PASSWORD}
+ENV DB_USERNAME ${DB_USERNAME}
+ENV DB_NAME ${DB_NAME}
+
+RUN mvn clean install -DskipTests=true
+
+EXPOSE 8080
+
+CMD ["java","-jar", "/code/target/java-dropwizard-flyway-starter-1.0-SNAPSHOT.jar", "server", "/code/config.yml"]
```

Comment:
is this correct? we renamed our project to kainos-job-role-manager, should we be expecting to use the old project name for the jar file?

Reply:
No we shouldn't, just changed to reflect new nam

Reply:
Search in the code for instances of java-dropwizard-flyway-starter (shift shift)

Reply:
Just updated